client/src/EntryList.jsx

```diff
@@ -0,0 +1,18 @@
+export default function Entries({ data, onClick, onEdit}) {
+  return (
+    <div>
+      <h3>Entries</h3>
+      <button onClick={onClick}>NEW</button>
+      <ul>
+        {data.entries.map((entry, index) => (
+          <li key={index}>
```

Comment:
The entryId would be an ideal value for the key, especially since the list can change as items are added/removed.